ci-operator/step-registry/stackrox/collector/integration-tests/stackrox-collector-integration-tests-workflow.yaml

```diff
@@ -5,7 +5,7 @@ workflow:
       - ref: stackrox-collector-integration-tests-setup
     test:
       - as: push-images
-        commands: .openshift-ci/jobs/push-images/push-images.sh
+        commands: .openshift-ci/jobs/push-images/push-images.sh collector
```

Comment:
I think this change might be outdated, I can't see where the first parameter is used in the script.